.github/workflows/build-test-e2e.yml

```diff
@@ -9,6 +9,9 @@ env:
   PW_ALL_BLOBS_DIR: all-blob-reports
 jobs:
   build:
+    # Run in Sage repo on develop or release- branches
+    # and on all branches in user-owned forks
+    if: ${{ github.ref_name == 'develop' || startsWith(github.ref_name, 'release-') || github.actor == github.repository_owner }}
```

Comment:
Move conditional to first job so that subsequent jobs are skipped when first job is skipped -- will properly skip the entire workflow for tagged branches, rather than running the first job, skipping the second, and then failing the last job, as in [475.0](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/6595041108)